packages/next-remoter/src/composable/useRouteBasedTools.ts

```
@@ -119,6 +114,7 @@ export function useRouteBasedTools(options: {
    */
   const syncAllRoutes = () => {
     // 功能关闭时：恢复之前被隐藏的工具，然后退出
+    debugger
```

Biome (2.4.6)

[error] This is an unexpected use of the debugger statement.

(lint/suspicious/noDebugger)